simplifierIntro.rst

```diff
@@ -1,40 +1,31 @@
 Introduction to Simplifier.net
 ==============================
 
-Simplifier.net is a FHIR registry. Within this registry you can
-create, upload, download, find and view FHIR Conformance Resources.
-Simplifier.net offers functionality for management of FHIR Resources
-and collaboration in teams. 
-
-FHIR (Fast Health Interoperability Resources) is an HL7 standard for information exchange in health care. It is based on Internet
-technology. More information about FHIR can be found at `HL7 FHIR <http://www.hl7.org/fhir/>`_.
-
-FHIR Profiling
---------------
-
-The FHIR Profiling tool `Forge (https://fire.ly/forge) <https://fire.ly/forge>`_ can be
-used off-line and independently from Simplifier.net.
+.. image:: ./images/Illustrations_Firely-03.png
+  :align: right
+  :width: 250px
+  :alt: Collaborating on a FHIR project
+  
+Simplifier is a FHIR registry, where you can create, upload, download, find an view FHIR resources. Simplifier is the best place to collaborate on FHIR projects with your team. 
 
 Simplifier accounts
 -------------------
 
-Simplifier.net distinguishes between free (personal) plans and paid
-plans. Anyone can find and download Resources in this Registry, with
-or without an account. Creating Resources requires an account. Paid
-plans are targeted at professional users and organizations.
+Simplifier offers everyone a free plan which allows you to find and download resources from the Simplifier FHIR registry. Each user can add one public project for free, making the user the owner of the project. For professional users and organizations we offer several paid plans which come with `additional functionality and benefits <https://simplifier.net/features/>`_ . Paid accounts are able to create more and private projects depending on the account level. 
```

Comment:
Shall we link this to https://simplifier.net/pricing instead? From there is another link to features if more convincing is needed.